microprofile/metrics-smallrye/src/main/resources/io/smallrye/metrics/base-metrics.properties

```diff
@@ -1,18 +1,18 @@
-classloader.currentLoadedClass.count.displayName: Current Loaded Class Count
-classloader.currentLoadedClass.count.type: counter
-classloader.currentLoadedClass.count.unit: none
-classloader.currentLoadedClass.count.description: Displays the number of classes that are currently loaded in the Java virtual machine.
-classloader.currentLoadedClass.count.mbean: java.lang:type=ClassLoading/LoadedClassCount
-classloader.totalLoadedClass.count.displayName: Total Loaded Class Count
-classloader.totalLoadedClass.count.type: counter
-classloader.totalLoadedClass.count.unit: none
-classloader.totalLoadedClass.count.description: Displays the total number of classes that have been loaded since the Java virtual machine has started execution.
-classloader.totalLoadedClass.count.mbean: java.lang:type=ClassLoading/TotalLoadedClassCount
-classloader.totalUnloadedClass.count.displayName: Total Unloaded Class Count
-classloader.totalUnloadedClass.count.type: counter
-classloader.totalUnloadedClass.count.unit: none
-classloader.totalUnloadedClass.count.description: Displays the total number of classes unloaded since the Java virtual machine has started execution.
-classloader.totalUnloadedClass.count.mbean: java.lang:type=ClassLoading/UnloadedClassCount
+classloader.loadedClasses.count.displayName: Current Loaded Class Count
+classloader.loadedClasses.count.type: gauge
```

Comment:
I presume this 'gauge' value is intentional?

Reply:
that's a change in the spec: https://github.com/eclipse/microprofile-metrics/blob/master/spec/src/main/asciidoc/required-metrics.adoc#classloading-jvm-stats